.gitignore

```diff
@@ -6,6 +6,7 @@
 
 # dependencies
 /node_modules/
+tests/dummy/lib/**/node_modules
```

Comment:
now that ember engines are symlinked to node_modules it ends up tracking node_modules underneath the dummy app folder. I changed this to be much more specific about exactly which node_modules we are excluding. For much of this setup I just followed whatever ember-engines repo is doing. I just checked and this is how it is handled there as well: https://github.com/ember-engines/ember-engines/blob/master/.gitignore#L17

Reply:
Gotcha! Thank you, great explanation and that makes perfect sense.